src/main/java/com/gregtechceu/gtceu/core/mixins/jei/FluidHelperMixin.java

```diff
@@ -2,25 +2,24 @@
 
 import com.gregtechceu.gtceu.client.TooltipsHandler;
 
-import net.minecraft.network.chat.Component;
 import net.minecraft.world.item.TooltipFlag;
 import net.minecraftforge.fluids.FluidStack;
 
+import mezz.jei.api.gui.builder.ITooltipBuilder;
 import mezz.jei.forge.platform.FluidHelper;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
-import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
-
-import java.util.List;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(FluidHelper.class)
 public class FluidHelperMixin {
 
-    @Inject(method = "getTooltip*", at = @At("TAIL"), remap = false)
-    private void gtceu$injectFluidTooltips(FluidStack ingredient, TooltipFlag tooltipFlag,
-                                           CallbackInfoReturnable<List<Component>> cir) {
-        var tooltip = cir.getReturnValue();
-        TooltipsHandler.appendFluidTooltips(ingredient.getFluid(), ingredient.getAmount(), tooltip, tooltipFlag);
+    @Inject(method = "getTooltip(Lmezz/jei/api/gui/builder/ITooltipBuilder;Lnet/minecraftforge/fluids/FluidStack;Lnet/minecraft/world/item/TooltipFlag;)V",
+            at = @At("TAIL"),
+            remap = false)
```

Comment:
@screret could you add a `require = 0` to the inject here so that if it fails to find the injection site, it fails without crashing?
Losing the ability to add info to the tooltip when JEI updates seems better than crashing, imo.